source/includes/gcp/_vpn_gateways.md

```diff
@@ -1 +1,189 @@
+curl -X GET \
+   -H "MC-Api-Key: your_api_key" \
+   "https://cloudmc_endpoint/v1/services/gcp/test-area/vpngateways/7559452436722963032"
+
+# Example:
+```
+
+```json
+{
+    "data": {
+      "creationTimestamp": "2019-12-17T06:56:55.724-08:00",
+      "description": "",
+      "region": "https://www.googleapis.com/compute/v1/projects/test-area/regions/northamerica-northeast1",
+      "network": "https://www.googleapis.com/compute/v1/projects/test-area/global/networks/default",
+      "status": "READY",
+      "selfLink": "https://www.googleapis.com/compute/v1/projects/test-area/regions/northamerica-northeast1/targetVpnGateways/your-vpngateway",
+      "forwardingRules": [
+        "https://www.googleapis.com/compute/v1/projects/test-area/regions/northamerica-northeast1/forwardingRules/your-forwarding-rule"
+      ],
+      "kind": "compute#targetVpnGateway",
+      "externalIp": {
+        "creationTimestamp": "2019-12-16T13:43:29.481-08:00",
+        "description": "",
+        "address": "35.203.1.33",
+        "prefixLength": 0,
+        "status": "IN_USE",
+        "region": "https://www.googleapis.com/compute/v1/projects/test-area/regions/northamerica-northeast1",
+        "selfLink": "https://www.googleapis.com/compute/v1/projects/test-area/regions/northamerica-northeast1/addresses/your-external-ip",
+        "users": [
+          "https://www.googleapis.com/compute/v1/projects/test-area/regions/northamerica-northeast1/forwardingRules/your-forwarding-rule"
+        ],
+        "networkTier": "PREMIUM",
+        "addressType": "EXTERNAL",
+        "kind": "compute#address",
+        "shortUsers": [
+          "your-forwarding-rule"
+        ],
+        "type": "STATIC",
+        "id": "1355427342211236974",
+        "name": "your-external=ip",
+        "shortRegion": "northamerica-northeast1"
+      },
+      "tunnelsName": [],
+      "reserveStaticIP": false,
+      "id": "7559452436722963032",
+      "name": "your-vpngateway",
+      "shortRegion": "northamerica-northeast1"
+    }
+}
+```
+
+<code>GET /services/<a href="#administration-service-connections">:service_code</a>/<a href="#administration-environments">:environment_name</a>/vpngateways/:id</code>
+
+Retrieve a VPN gatewau in an [environment](#administration-environments).
+
+| Attributes                       | &nbsp; |
+| -------------------------------- | ------ |
+| `creationTimestamp`<br/>*string* | Creation timestamp in RFC3339 text format. |
+| `region`<br/>*string*            | The self link of the region |
+| `network`<br/>*string*           | The self link of the network |
+| `status`<br/>*string*            | The status of the this resource |
+| `selfLink`<br/>*string*          | Server-defined URL for the resource. |
+| `forwardingRules`<br/>*List<Objects>*  | The forwarding rules that this resource attached to |
+| `kind`<br/>*string*              | Type of the resource. |
+| `externalIp`<br/>*Object*        | The external IP this resource is attached to |
```

Comment:
is there a description of of the externalip object... maybe we should have a link to it?

Reply:
fd3c96eba72d287295e385d0a0a8571ec858bd18